cybergis_compute_client/CyberGISCompute.py

```diff
@@ -144,21 +144,20 @@ def login_json(self):
         """
         Checks for json file and calls login_token function.
         """
-        with open(os.path.abspath('cybergis_compute_user.json')) as f:
-            user = json.load(f)
-            token = user['token']
-            print('📃 Found "cybergis_compute_user.json"')
-            print('NOTE: if you want to login as another user, please remove this file')
-            try:
-                self.jupyterhubApiToken = token
-                self.set_username()
-                self.save_token()
-                return self.login()
-            except:
-                envToken = os.getenv('JUPYTERHUB_API_TOKEN')
-                if envToken is not None:
-                    return self.host_token_login(envToken)
-                print('❌ Failed to login via token JSON file')
+        try:
+            with open(os.path.abspath('cybergis_compute_user.json')) as f:
+                user = json.load(f)
+                token = user['token']
+            print('📃 Found "cybergis_compute_user.json! NOTE: if you want to login as another user, please remove this file')
+            self.jupyterhubApiToken = token
+            self.set_username()
+            self.save_token()
+            return self.login()
+        except:
+            # print('❌ Failed to login via token JSON file, trying environment variable...')
```

Comment:
Is there a reason why this is commented ?

Reply:
Yeah I commented it out for now because I was worried that if an average user saw it, they'd think something was broken and report an error. However, it's an extremely helpful statement for debugging so I didn't want to delete it.

Reply:
I see got it. One other thing I realized was that the invalid json file gets overwritten by the envToken through https://github.com/cybergis/cybergis-compute-python-sdk/blob/8fdc406735e5c3d7a30dc5ca4b81aa03101b5f4a/cybergis_compute_client/CyberGISCompute.py#L113. Does it make sense to have some kind of print statement letting the user know about this so that they know that their original token was invalid ?

Reply:
I don't think it really makes sense to let the user know their original JSON was invalid because the user probably doesn't even know about the JSON or what it does to begin with.